### PR TITLE
Add question converters under tools directory

### PR DESCRIPTION
**Pull request by coatless**
This PR takes previously written question converters for various LMS systems into the `tools` directory with extended documentation. 

In particular, there is now:

- randexam (via e-mail)
- loncapa (via repository with kebob case changes)
- smartPhysics (via #2167)

Once merged, the [`question-converters`](https://github.com/PrairieLearn/question-converters) repository can be deleted.

### CODE REVIEW

File: tools/lms-question-converters/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Convert Learning Management System Questions to PrairieLearn
+
+Within this directory are a set of conversion tools for moving previously written questions
+in a different Learning Management System (LMS) to the question format for PrairieLearn. 
+
+- Craig Zilles: [LON-CAPA](lon-capa/)
+- Lawrence Angrave: [randexam](randexam/)
+- Lucas Wagner: [smartPhysics (FlipIt Physics)](smart-physics/)
```
Comment (mwest1066) [inline tools/lms-question-converters/README.md]: Let's not associate people's names with these. We don't do it elsewhere in the code. I'm fine to add names to the commit notes or somewhere like that. If we want to get really creative, you could actually make commits with these people as authors for each individual converter.

File: tools/lms-question-converters/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## Convert Learning Management System Questions to PrairieLearn
```
Comment (mwest1066) [inline tools/lms-question-converters/README.md]: Maybe just `question_converters` as a directory name? We've been a bit inconsistent with snake_case versus kebab-case, but `node_modules` is probably a good reference.

Comment (coatless) [reply tools/lms-question-converters/README.md]: Done.

File: tools/lms-question-converters/README.md (hunk 1 above)
Comment (coatless) [reply tools/lms-question-converters/README.md]: Done. Switched the committers to Craig and Lawerence.

**Pull request merged**